vispy/visuals/filters/color.py

```diff
@@ -180,10 +185,5 @@ def __init__(self, cmap, zrange=(0, 1)):
         self.fshader['zval'] = self.vshader['zval']
 
     def _attach(self, visual):
-        self._visual = visual
-        vhook = visual._get_hook('vert', 'post')
-        vhook.add(self.vshader(), position=9)
-        fhook = visual._get_hook('frag', 'post')
-        fhook.add(self.fshader(), position=3)
-
+        super(ZColorMapFilter, self)._attach(visual)
```

Comment:
`ZColormapFilter` was used above.

Reply:
Just because it took me more than 5 seconds to figure out what you were saying: the `m` in the class name should be lowercase.